concent_api/core/views.py

```diff
@@ -617,6 +618,16 @@ def deserialize_message(raw_message_data):
         raise Http400("Unable to deserialize Golem Message: {}.".format(exception))
 
 
+def validate_report_computed_task_time_window(report_computed_task):
+    if not isinstance(report_computed_task, message.ReportComputedTask):
```

Comment:
In `handle_send_force_report_computed_task` you are already after passing this condition in `send()`, so I think only `assert` should be here.